### PR TITLE
chore: Adds repo owners and accessibility sections

### PR DESCRIPTION
**Pull request by marcabisamra**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -141,28 +141,47 @@ We will strive to make generic tests that will not break during refactors or Dat
...
-	_ How many contributors, is there a company backing the project, or a whole community.
+	_ The last commit was within one year to date
+	_ How many contributors, is there a company backing the project, or a whole community
 	_ Don't pull in branches contributed by only one developer
 	_ How many open PRs? and is anybody reviewing them
-- Is it stable? - How many open issues and how long do issues take before they are fixed.
+- Is it stable? - How many open issues and how long do issues take before they are fixed
 - How many downloads, we want to find packages that have at least a couple thousand downloads
 - follows semantic versioning MAJOR.MINOR.PATCH (e.g. 2.5.1)
 	_ our `package.json` file should use the `^` will update the package to the latest patch and minor version.
```
Comment (MechanicalHuman) [inline CONTRIBUTING.md:168]: We should use `~` not `^`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,18 +20,18 @@ The following table explains how we interpret each of the states for a Pivotal s
 
 ## Git Workflow
 
-- We use [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/).
+- We use [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)
```
Comment (MechanicalHuman) [inline CONTRIBUTING.md:23]: It is git-flow tho? We don't have a `development` branch and do PRs directly to main. 

I do think we should have a dev branch

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -141,28 +141,47 @@ We will strive to make generic tests that will not break during refactors or Dat
...
+- Include alternative text for images
+- Identify page language and language changes
+- Use mark-up to convey meaning and structure
+- Help users avoid and correct mistakes
+- Reflect the reading order in the code order
+- Write code that adapts to the user’s technology
+- Provide meaning for non-standard interactive elements using `aria`
+- Ensure that all interactive elements are keyboard accessible
+- Avoid CAPTCHA where possible
+- Use color blind accessible colors!
```
Comment (MechanicalHuman) [inline CONTRIBUTING.md]: Everyone peddling their agendas...

**Review by MechanicalHuman — COMMENTED** [review]
👁‍🗨👁‍🗨👁‍🗨

File: CONTRIBUTING.md (hunk 3 above)
Comment (marcabisamra) [reply CONTRIBUTING.md]: 😏

File: CONTRIBUTING.md (hunk 1 above)
Comment (marcabisamra) [reply CONTRIBUTING.md:168]: Yes!

File: CONTRIBUTING.md (hunk 2 above)
Comment (marcabisamra) [reply CONTRIBUTING.md:23]: fair!!

**Review by mattfelsen — APPROVED** [review]
💙

**Pull request merged**